Source/gravity/Gravity.cpp

```diff
@@ -2927,6 +2927,19 @@ Gravity::add_pointmass_to_gravity (int level, MultiFab& phi, MultiFab& grav_vect
             Real y = problo[1] + (static_cast<Real>(j) + 0.5_rt) * dx[1] - problem::center[1];
             Real z = problo[2] + (static_cast<Real>(k) + 0.5_rt) * dx[2] - problem::center[2];
 
+            if(castro::point_mass_offset_is_true == 1){
+                Real star_radius = castro::point_mass_location_offset;
+
+            if(AMREX_SPACEDIM == 1){
+                x += star_radius;
```

Comment:
can you fix this indenting?  right now it appears like this if block is not nested inside the one above.